docs/source/user/build_manually.rst

```diff
@@ -274,6 +274,105 @@ Prepare:
     git clone https://github.com/trilinos/Trilinos.git
     cd $nalu_build_dir/packages/Trilinos
     mkdir build
+    cd build
+
+Now create a ``do-configTrilnos`` script with the following recommended options:
+
+.. code-block:: bash
```

Comment:
Recommend we change this to `.. literalinclude::` and then also provide a link for the user to download the file.